qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -38,31 +38,52 @@
-    and corresponding durations.
+    r"""
+    A piecewise-constant driven control for a single qubit.
 
     Parameters
     ----------
     rabi_rates : np.ndarray, optional
-        1-D array of size nx1 where n is number of segments;
-        Each entry is the rabi rate for the segment. Defaults to None.
+        The Rabi rates :math:`\{\Omega_n\}` for each segment, in units of radians per second. Every
+        element must be non-negative. Represented as a 1D array of length :math:`N`, where :math:`N`
+        is number of segments. You can omit this field if the Rabi rate is zero on all segments.
     azimuthal_angles : np.ndarray, optional
-        1-D array of size nx1 where n is the number of segments;
-        Each entry is the azimuthal angle for the segment; Defaults to None.
+        The azimuthal angles :math:`\{\phi_n\}` for each segment. Represented as a 1D array of
+        length :math:`N`, where :math:`N` is number of segments. You can omit this field if the
+        azimuthal angle is zero on all segments.
     detunings : np.ndarray, optional
-        1-D array of size nx1 where n is the number of segments;
-        Each entry is the detuning angle for the segment; Defaults to None.
+        The detunings :math:`\{\Delta_n\}` for each segment, in units of radians per second.
+        Represented as a 1D array of length :math:`N`, where :math:`N` is number of segments. You
+        can omit this field if the detuning is zero on all segments.
     durations : np.ndarray, optional
-        1-D array of size nx1 where n is the number of segments;
-        Each entry is the duration of the segment (in seconds); Defaults to None.
+        The durations :math:`\{\delta t_n\}` for each segment, in units of seconds. Every element
+        must be positive. Represented as a 1D array of length :math:`N`, where :math:`N` is number
+        of segments. Defaults to an array of ones if omitted.
     name : string, optional
-        An optional string to name the driven control. Defaults to None.
+        An optional string to name the control. Defaults to ``None``.
 
     Raises
     ------
     ArgumentsValueError
         Raised when an argument is invalid.
+
+    Notes
+    -----
+    This class represents a control for a single driven qubit with Hamiltonian:
+
+    .. math::
+
+        H(t) = \frac{1}{2}\left(\Omega(t) e^{i\phi(t)} \sigma_- +
+                                \Omega(t) e^{-i\phi(t)}\sigma_+\right) +
+               \frac{1}{2}\Delta(t)\sigma_z,
+
+    where :math:`\Omega(t)` is the Rabi rate, :math:`\phi(t)` is the azimuthal angle (or drive
+    phase), :math:`\Delta(t)` is the detuning, :math:`\sigma_\pm = (\sigma_x \mp \sigma_y)/2`,
+    and :math:`\sigma_k` are the Pauli matrices.
+
+    The controls are piecewise-constant, meaning :math:`\Omega(t)=\Omega_n` for
+    :math:`t_{n-1}\leq t<t_n`, where :math:`t_0=0` and :math:`t_n=t_{n-1}+\delta t_n` (and similarly
+    for :math:`\phi(t)` and :math:`\Delta(t)`).
```

Comment:
nit: would it help to add the unitary and total rotation from the Hamiltonian? (see https://docs.q-ctrl.com/wiki/control-operations#single-qubit-driven).
It might be too much for the note (and that's why it is sad to see wiki go)

Reply:
I've added a section, let me know what you think